lib/daylight.cpp

```diff
@@ -840,6 +859,9 @@ bool Daylight::simStandard(int blindGroupNum, int setting, Control *model){
         sunsOct=model->spaceDirectory()+model->intermediateDataDirectory()+model->spaceName()+"_"+model->windowGroups()[blindGroupNum].name()+"_sun_set"+std::to_string(setting+1)+"_std.oct";
 
     }
+    if (writeCL){
+            std::cout<<"***\nCreating suns octree.\n***"<<std::endl;
```

Comment:
I'm not sure that these messages are a great idea. Can we integrate them into the logging setup that we have? If we need to add a "debug" message type, we can do that.